### PR TITLE
some updates

### PR DESCRIPTION
**Pull request by PrajwalPrashanth**
- [x] notify with log_records
- [x] Change jupyter extension logo
- []

### CODE REVIEW

File: jovian/utils/records.py [hunk 1]
```diff
@@ -59,11 +60,14 @@ def log_record(record_type, data=None, verbose=True, **data_args):
     tracking_slug = res['tracking']['trackingSlug']
     # Save to data block
     _data_blocks.append((tracking_slug, record_type, data))
+
+    if notify:
```
Comment (siddhantujjain) [inline jovian/utils/records.py]: how about doing this from server directly? You can pass notify param along with log_* calls

Comment (PrajwalPrashanth) [reply jovian/utils/records.py]: Let's have discussion and do this in a next PR. Currently merging this for updated logo

Comment (siddhantujjain) [reply jovian/utils/records.py]: you can create a separate PR for updated logo

Comment (PrajwalPrashanth) [reply jovian/utils/records.py]: Done

**Comment by aakashns** [comment]
is this ready for merge?

**Comment by aakashns** [comment]
What's the status of this PR?

**Comment by PrajwalPrashanth** [comment]
Closing this PR, will look at notify with log_*() later

**Pull request closed without merging**